.pre-commit-config.yaml

```diff
@@ -30,7 +30,7 @@ repos:
         args: [ "--config=./tools/pyproject.toml", "--fix" ]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v1.3.0
+    rev: v1.4.0
```

Comment:
Was trying to address local-vs-GHA mypy error diffs.  While this didn't help, seems reasonable to keep the updated versions.

Reply:
You (all) may want to periodically run `pre-commit autoupdate`.  Thanks for doing this manually